pkg/helm/actions/upgrade_release_test.go

```diff
@@ -0,0 +1,98 @@
+package actions
+
+import (
+	"errors"
+	"io/ioutil"
+	"testing"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	"helm.sh/helm/v3/pkg/time"
+)
+
+func TestUpgradeRelease(t *testing.T) {
+	tests := []struct {
+		releaseName  string
+		chartPath    string
+		chartName    string
+		chartVersion string
+		err          error
+	}{
+		{
+			releaseName:  "upgrade valid release should return successful response",
```

Comment:
this should be a field called `testName`, not `releaseName`?

Reply:
Done